examples/nestjs-provider/test/pact/pact-provider-config-options.service.ts

```diff
@@ -28,24 +28,24 @@ export class PactProviderConfigOptionsService
           this.animalRepository.clear();
           token = '1234';
 
-          return 'Animals removed to the db';
+          return { description: 'Animals removed to the db' };
```

Comment:
what is the purpose of this change?

Reply:
Tests were failing due to the need of having JsonMap as a return type. Also showing that imports didn't resolve correctly.

Reply:
This shouldn't break with the change, though - because nestjs-pact only uses the v2 options type, which hasn't changed.

Reply:
Really these tests shouldn't be returning anything at all, though - because I don't think it's valid in V2. @mefellows, thoughts?

Reply:
Correct. I actually think it's completely misleading, so would prefer to remove altogether.